meta-aws-demos/conf/devices/agl-nxp-goldbox/layers.conf

```diff
@@ -26,4 +26,8 @@ BBLAYERS = "\
     ${OEROOT}/../layers/agl/meta-agl-demo \
```

Comment:
Is the agl-nxp-goldbox with ros building successfully?

Reply:
Yes